akka-distributed-data/src/main/scala/akka/cluster/ddata/Replicator.scala

```diff
@@ -1953,13 +1953,22 @@ final class Replicator(settings: ReplicatorSettings) extends Actor with ActorLog
   def receiveDeltaPropagationTick(): Unit = {
     deltaPropagationSelector.collectPropagations().foreach {
       case (node, deltaPropagation) =>
-        // TODO split it to several DeltaPropagation if too many entries
-        if (deltaPropagation.deltas.nonEmpty)
+        if (deltaPropagation.deltas.nonEmpty) {
+          deltaPropagation.deltas.foreach {
+            case (key, entries) => log.debug(
+              "Sending DeltaPropagation from [{}] for [{}] with sequence numbers [{}], current version is [{}]",
+              deltaPropagation.fromNode,
+              key,
+              entries.fromSeqNr + "-" + entries.toSeqNr,
+              deltaPropagationSelector.currentVersion(key)
+            )
+          }
           replica(node) ! deltaPropagation
+        }
     }
 
-    if (deltaPropagationSelector.propagationCount % deltaPropagationSelector.gossipIntervalDivisor == 0)
-      deltaPropagationSelector.cleanupDeltaEntries()
+//    if (deltaPropagationSelector.propagationCount % deltaPropagationSelector.gossipIntervalDivisor == 0)
```

Comment:
we don't need to clean delta as often, because gossipIntervalDivisor is not used now to calculate delta replication interval